test/lib/ansible_test/_data/requirements/units.txt

```diff
@@ -50,3 +50,7 @@ openshift ; python_version >= '2.7'
 
 # requirement for maven_artifact
 semantic_version
+
+# requirement for docker_login
+docker ; python_version >= '2.7'
```

Comment:
I'm not sure whether that's acceptable. In general, unit tests are supposed to mock libraries they are using. (Otherwise, the testing container would be swamped in a large amount of random and potentially incompatible libraries :) )